src/API/Loggy.Api/Loggy.Api.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.0" />
+    <PackageReference Include="MongoDB.Driver" Version="2.7.3" />
   </ItemGroup>
 
 </Project>
```

Comment:
Just a thought on this - would it not maybe make sense to have a Loggy.Data (DataAccess / Db ?) Project, and have that project have a nice clean interface, and it in turn references some sort of Loggy.Data.Mongo project?  And of course then that Loggy.Data.Mongo project references Mongo specific nuget packages. 

I think that might be cleaner and it gives us flexibility with the scenario where we decide "actually mongo has some issues and so we want to switch to ... sql (for example)" 

What are your thoughts @willemodendaal ?

Reply:
Sure, that's worth considering. I've felt pains with too-tight coupling to the db layer before. Looking into this now.

Reply:
coolbeans :) looking forward to seeing how it goes